packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -475,12 +475,22 @@ export const FocusableCells: TableStory = {
   name: 'focusable cells'
 };
 
-let manyColunns = [];
+interface Iitem {
+  name: string,
+  key: string
+}
+
+let manyColunns: Array<Iitem> = [];
 for (let i = 0; i < 100; i++) {
   manyColunns.push({name: 'Column ' + i, key: 'C' + i});
 }
 
-let manyRows = [];
+interface Irow {
```

Comment:
same. sorry, idk where this naming convention came from. `RowValue`, `ExampleRow`, etc.?